app/service_invite/rest.py

```diff
@@ -223,3 +223,14 @@ def validate_service_invitation_token(token):
 
     invited_user = get_invited_user_by_id(invited_user_id)
     return jsonify(data=invited_user_schema.dump(invited_user)), 200
+
+
+@service_invite.route("/service/invite/redis/<redis_key>", methods=["GET"])
+def get_redis_data(redis_key):
```

Comment:
Actually, one quick question - should we make this a more general API method for retrieving any Redis key needed from the API, or should this method name be more consistent with what the route is set up for?

Reply:
I went back and forth on that.  Ultimately I decided for consistency that it should be one API call for each redis retrieval, so we don't end up with one API and a method that is 5000 lines like with 500 if/elif statements.

However, the method name is a bit generic, I will fix that.  I'm going add a different API for organization invites, etc.